src/wrapper-start.js

```diff
@@ -37,18 +37,11 @@
     var newErr;
     if (err instanceof Error) {
       var newErr = new Error(err.message, err.fileName, err.lineNumber);
-      if (isBrowser) {
-        newErr.message = err.message + '\n\t' + msg;
-        newErr.stack = err.stack;
-      }
-      else {
-        // node errors only look correct with the stack modified
-        newErr.message = err.message;
-        newErr.stack = err.stack + '\n\t' + msg;
-      }
+      newErr.message = msg + '\n\t' + err.message;
```

Comment:
My thought was that the *message* was clearer as increasing specificity from left to right. "There was and error" "during loading" "this file" "which you called this" "from this caller" "caused by a missing file"

I completely agree that the *stack* should be in that order.

That said, while I think the increasing specificity of the message has clear merit, I'm not attached to it. I can swap it back if preferred.